app/views/pages/GUIDE.markdown

```diff
@@ -43,63 +43,64 @@ Please add yourself to this list (link is optional) if you've contributed to thi
 | COMP-3520 | Introduction to Computer Graphics | Not Offered | Offered | Not Offered |  |
-| COMP-3540 | Theory of Computation | Offered | Not Offered | Not Offered |  |
-| COMP-3670 | Computer Networks | Offered | Not Offered | Offered |  |
-| COMP-3710 | Artificial Intelligence Concepts | Not Offered | Offered | Not Offered |  |
+| COMP-3540 | Theory of Computation | Offered | Not Offered | Not Offered | 💙💚 |
+| COMP-3670 | Computer Networks | Offered | Not Offered | Offered | 💙💛💜 |
+| COMP-3710 | Artificial Intelligence Concepts | Not Offered | Offered | Not Offered | 💚 |
 | COMP-3770 | Game Design, Development, and Tools | Offered | Not Offered | Not Offered |  |
-| COMP-4110 | Software Verification and Testing | Not Offered | Offered | Not Offered |  |
-| COMP-4150 | Advanced and Practical Database Systems | Offered | Not Offered | Not Offered |  |
-| COMP-4200 | Mobile Application Development | Offered | Not Offered | Not Offered |  |
-| COMP-4220 | Agile Software Development | Offered | Not Offered | Not Offered |  |
-| COMP-4250 | Big Data Analytics and Database Design | Not Offered | Offered | Not Offered |  |
-| COMP-4400 | Principles of Programming Languages | Offered | Not Offered | Offered |  |
+| COMP-4110 | Software Verification and Testing | Not Offered | Offered | Not Offered | 💚 |
+| COMP-4150 | Advanced and Practical Database Systems | Offered | Not Offered | Not Offered | 💛 |
+| COMP-4200 | Mobile Application Development | Offered | Not Offered | Not Offered | 💛 |
+| COMP-4220 | Agile Software Development | Offered | Not Offered | Not Offered | 💛 |
+| COMP-4250 | Big Data Analytics and Database Design | Not Offered | Offered | Not Offered | 💛 |
+| COMP-4400 | Principles of Programming Languages | Offered | Not Offered | Offered | 💙💚 |
 | COMP-4500 | Multimedia System Development | Not Offered | Offered | Not Offered |  |
-| COMP-4540 | Design and Analysis of Algorithms | Not Offered | Offered | Not Offered |  |
+| COMP-4540 | Design and Analysis of Algorithms | Not Offered | Offered | Not Offered | 💙💚 |
 | COMP-4670 | Network Security | Not Offered | Offered | Not Offered |  |
 | COMP-4700 | Project using Selected Tools | Offered | Not Offered | Not Offered |  |
 | COMP-4730 | Advanced Topics in AI I | Offered | Not Offered | Not Offered |  |
 | COMP-4740 | Advanced Topics in AI II | Not Offered | Offered | Not Offered |  |
 | COMP-4750 | Selected Topics | Not Offered | Offered | Not Offered |  |
 | COMP-4770 | Artifical Intelligence for Games | Not Offered | Offered | Not Offered |  |
-| COMP-4800 | Selected Topics in Software Engineering | Not Offered | Offered | Not Offered |  |
-| COMP-4960 | Research | Offered | Offered | Not Offered |  |
-| COMP-4990 | Project Management: Techniques and Tools | Offered | Offered | Not Offered |  |
+| COMP-4800 | Selected Topics in Software Engineering | Not Offered | Offered | Not Offered | 💚 |
+| ❗COMP-4960 | Research | Offered | Offered | Not Offered | 💙💚 |
```

Comment:
Don't worry about adding this and the "one or the other" notes, anyone who's in 4th year should know this and it ruins the prettiness 😛